README.md

```diff
@@ -13,6 +13,7 @@ Please ensure the following:
 
 Follow these steps:
   1.  Click the green ["use this template"](https://github.com/tc39/template-for-proposals/generate) button in the repo header. (Note: Do not fork this repo in GitHub's web interface, as that will later prevent transfer into the TC39 organization)
+  1. Update the biblio to the latest version: `npx npm-check-updates -u @tc39/ecma262-biblio`.
```

Comment:
why use npx here?
```suggestion
  1. Update the biblio to the latest version: `npm install --save-dev @tc39/ecma262-biblio@latest`.
```

Reply:
Because I didn't realize you could use `@latest` to override the semver constraint. Pushed. (It also needs `--save-exact` or npm will helpfully insert the `^`, alas.)